### PR TITLE
Added note creation UI

### PR DESCRIPTION
**Pull request by aagneyanNamboodiri**
Added the note creation UI with the new fields.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -48,8 +52,25 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
...
+                placeholder="Select Contact"
+              />
+              <Select
+                required
+                className="pt-4"
+                label="Tags"
+                name="tagList"
+                options={TAG_OPTIONS}
+                placeholder="Select tag"
+              />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:72]: Why is that in your business logic you are not allowing the user to add multiple tags to a note? We should be allowed to choose multiple tags.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 2]
```diff
@@ -48,8 +52,25 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
...
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
             />
+            <div className="w-full flex-col justify-between">
+              <Select
+                required
+                label="Assigned Contact"
+                name="contactList"
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:61]: Suffixing with the `List` keyword is unnecessary. Rather name this as `contacts`. But here, you are only selecting one contact. Thus having the `List` was wrong in the first place. Here, singular, `contact` should be the name.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 3]
```diff
@@ -1,5 +1,57 @@
 import * as yup from "yup";
 
+const CONTACTS_LIST = ["Oliver Smith", "Bilbo Baggins", "Frodo Baggins"];
+const TAGS_LIST = [
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:4]: Stop suffxing redundant keywords.

**Review by yedhink — COMMENTED** [review]
@aagneyanNamboodiri

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 1 above)
Comment (aagneyanNamboodiri) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:72]: I was expecting each note to be under one tag. 
I have added the functionality to chose multiple tags.

**Pull request merged**